Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -801,6 +801,9 @@ void AsyncScrollingCoordinator::scrollableAreaScrollbarLayerDidChange(Scrollable
 ScrollingNodeID AsyncScrollingCoordinator::createNode(ScrollingNodeType nodeType, ScrollingNodeID newNodeID)
 {
     LOG_WITH_STREAM(ScrollingTree, stream << "AsyncScrollingCoordinator::createNode " << nodeType << " node " << newNodeID);
+    // TODO: Need a better way to fix scrolling tree in iframe process
```

Comment:
Ideally file a bug to track the fix and reference it here.

Reply:
Added.